### ISSUE TITLE
[DOC] Clearer statement of primarily features

### ISSUE THREAD

**Issue by oesteban** [poldracklab/fmriprep#1215]
Following @TimVanMourik's open review comment (http://preprint.space/openreviewers/0000-0002-8155-8641), fMRIPrep should describe its primarily features more clearly and in detail:

- Automatic data type recognition
- Rich support of data types
- Rich visual performance feedback
- Increased performance by best-in-breed approach.

**Comment by emdupre** [comment]
It seems that this would, at least in part, overlap with the [Contributing to fMRIPrep](http://fmriprep.readthedocs.io/en/latest/contributors.html) section of the documentation, no?

**Comment by oesteban** [comment]
Yep, this would extend that section with more theoretical issues.

**Comment by oesteban** [comment]
I think this should be addressed now more generally in the nipreps documentation - moving there.
